hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -430,6 +430,12 @@ private ResponseCodeEnum resolvedStatusOfSubmission(HapiSpec spec) throws Throwa
         Query receiptQuery = txnReceiptQueryFor(extractTxnId(txnSubmitted));
         do {
             Response response = statusResponse(spec, receiptQuery);
+            if (response.getTransactionGetReceipt()
+                    .getHeader()
+                    .getNodeTransactionPrecheckCode()
+                    .equals(ResponseCodeEnum.PLATFORM_NOT_ACTIVE)) {
+                throw new IllegalStateException("Node is not active");
+            }
```

Comment:
Does it not throw on client side now ?

Reply:
There is no exception thrown when platform returning PLATFORM_NOT_ACTIVE. I added it here so it will go up to one of Throwable in this method to the calling method. This helped me debugging the issue. Not sure if we really need this on mono code.

Reply:
Since this code is being used for mono-service also, I wonder if this breaks anything else

Reply:
Ok, I removed this change. Thanks.